src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -245,12 +263,19 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     toTime: Date,
     maxCount: number = 1,
     offset: number = 0,
+    reqConfig: RequestConfiguration,
     maxCloudCoverPercent?: number | null,
     datasetParameters?: Record<string, any> | null,
   ): Promise<{ data: { tiles: any[]; hasMore: boolean } }> {
     if (!searchIndexUrl) {
       throw new Error('This dataset does not support searching for tiles');
     }
+    if (!maxCount) {
+      maxCount = 1;
+    }
+    if (!offset) {
+      offset = 0;
+    }
```

Comment:
To make it clearer what is happening, I would suggest changing the default values to `null` and checking explicitly for `null`, not just falsey:
```
    maxCount: number | null = null,
    offset: number | null = null,
...
    if (maxCount === null) {
      maxCount = 1;
    }
    if (offset === null) {
      offset = 0;
    }
```

Reply:
Sure, that's better, since passing undefined does not affect the default values.

Reply:
This is not what I meant, sorry if I wasn't clear. I don't think the example in README works right now. I think these changes are needed:
- example in README should use `null` instead of `undefined` for `findTiles` parameters `maxCount` and `offset`
- definition of `maxCount` and `offset` parameters should be changed as above (`null` is allowed and is a default value) - in **every** definition of `findTiles`, and especially in `AbstractLayer`:
  ```
  maxCount: number | null = null,
  offset: number | null = null,
  ```
- default value for `maxCount` needs to be consistent; since it was `50` before (see `AbstractLayer`), I would suggest we keep `50`: 
  ```javascript
  if (maxCount === null) {
    maxCount = 50;
  }
  ```